### PR TITLE
Create slider for dates

### PR DESCRIPTION
**Pull request by coconut750750**
- Add a slider component

- When a tract is selected, the dates and rates are queries from the backend. The DateSlider class uses the dates to construct a slider, and the rates will be displayed on a graph above the slider, and the tract colors on the map will change.

ezgif-2-995eeb0d2c6f

### CODE REVIEW

File: frontend/src/components/DateSlider.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Slider from 'rc-slider';
+import 'rc-slider/assets/index.css';
+
+// const wrapperStyle = { width: 200, margin: 0 };
```
Comment (meghatron3000) [inline frontend/src/components/DateSlider.js]: delete

**Review by meghatron3000 — APPROVED** [review]

File: frontend/src/components/DateSlider.js (hunk 1 above)
Comment (meghatron3000) [reply frontend/src/components/DateSlider.js]: Looks good to me!

File: frontend/src/components/Graph.js [hunk 2]
```diff
@@ -49,9 +50,11 @@ class Graph extends React.Component {
     let xLabelList = [rates_list[0]["x"]];
 
     for (let i = 1; i <= STEPS; i++) {
-      xLabelList.push(iterator + xLabelList[i - 1]);
+      xLabelList.push(iterator + parseInt(xLabelList[i - 1]));
     }
 
+    console.log(xLabelList);
```
Comment (yousefa00) [inline frontend/src/components/Graph.js]: don't forget logs

File: frontend/src/components/Home.js [hunk 3]
```diff
@@ -33,7 +37,8 @@ class Home extends React.Component {
       tractSelected: false,
       currentTract: {},
       geocoderValue: "",
-      displayGraph: false
+      displayGraph: false,
+      selectedDate: 25
```
Comment (caiyingshen) [inline frontend/src/components/Home.js:41]: not sure if this is the best value for default selected date, if there's some way to automatically find the latest recorded response rate that would be better

**Review by caiyingshen — APPROVED** [review]
is this allowed lol

**Pull request merged**